src/astf/astf_proto.h

```diff
@@ -0,0 +1,56 @@
+#ifndef ASTF_PROTO_H
```

Comment:
why do you need this file?

Reply:
I'm sorry it is added abruptly. I'll remove it.